front-end/src/Home.js

```diff
@@ -4,20 +4,22 @@ import Footer from './Footer'
 import './Home.css'
 
 function Home() {
-    return (
-	<>
-		<Header/>
+	return (
+		<>
+			<Header />
+			<fieldset class="home">
```

Comment:
The changes you made to Home look good. Do you think you could also remove the white border that surrounds each fieldset? Thanks.